src/chrome/content/rules/Nextag.xml

```diff
@@ -1,50 +1,52 @@
 <!--
-	For problematic rules, see Nextag-self-signed.xml.
-
-
-	Nonfunctional:
-
-		- (www.)nextag.es
-		- (www.)nextag.co.jp
-		- (www.)nextag.com.au
-
+	Mismatch:
+		img.nextag.com	( equal to imgsrv )
+		stores.nextag.com
+		tickets.nextag.com
+
+		www.nextag.ca
+		www.nextag.co.jp
+
+	Different http/https content:
+		www.nextag.com
+		www42.nextag.com
+		merchants.nextag.com
+		travel.nextag.com
+
+		www.nextag.com.au
+		www.nextag.de
+		www.nextag.fr
+		www.nextag.it
```

Comment:
There is also ^ and a m. subdomain, but I could not reproduce "Different http/https content".